writer_test.go

```diff
@@ -1225,6 +1225,27 @@ func ExampleMediaPlaylist_Segments_scte35_oatcls() {
 	// media2.ts
 }
 
+func ExampleMediaPlaylist_Segments_scte35_cue() {
+	f, _ := os.Open("sample-playlists/media-playlist-with-cue-scte35.m3u8")
+	p, _, _ := DecodeFrom(bufio.NewReader(f), true)
+	pp := p.(*MediaPlaylist)
+	fmt.Print(pp)
+	// Output:
+	// #EXTM3U
+	// #EXT-X-VERSION:3
+	// #EXT-X-MEDIA-SEQUENCE:0
+	// #EXT-X-TARGETDURATION:10
+	// #EXT-X-CUE-OUT:15
+	// #EXTINF:8.844,
+	// media0.ts
+	// #EXT-X-CUE-OUT-CONT:ElapsedTime=8.844,Duration=15
+	// #EXTINF:6.156,
+	// media1.ts
+	// #EXT-X-CUE-IN
+	// #EXTINF:3.844,
+	// media2.ts
```

Comment:
I don't think this comment adds any value, since the function only loads a fixture file, and we can just open the file to see the contents.

Reply:
I added this to have consistency with the other tests but I agree with you that we shouldn't be using prints. Not sure how it is done but somehow the function is testing that the print matches the comment. I will remove the print and test it properly.

Reply:
Fixed in: https://github.com/zencoder/m3u8/pull/8/commits/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0